### ISSUE TITLE
Missing fmi3 headers in CMAKE installed package

### ISSUE THREAD

**Issue by yoerg** [modelon-community/fmi-library#66]
I'm creating a binary package from _3.0a1_ with
````sh
cmake ../fmi-library-3.0a1
cmake --build .
cmake --install .
````
But the resulting `../install` directory is missing `FMI3/fmi3_xml_dimension.h` and possibly other FMI3 headers.

Trying to compile a source file containing `#include "fmilib.h"` with e.g. `gcc -I ../install -c source.c` results in an error:
````
In file included from ../install/include/FMI3/fmi3_import_variable.h:30,
                 from ../install/include/FMI3/fmi3_import.h:36,
                 from ../install/include/fmilib.h:30,
                 from source.c:1:
../install/include/FMI3/fmi3_import_dimension.h:30:10: fatal error: FMI3/fmi3_xml_dimension.h: No such file or directory
   30 | #include "FMI3/fmi3_xml_dimension.h"
````

I believe either the header is simply missing from `FMIIMPORT_PUBHEADERS` or the `#include` is unintentional.

Tested with _Unix Makefiles_ and _MSVC 19_.

**Comment by PeterMeisrimelModelon** [comment]
We have an outstanding ticket to fix this include issue for 3.0a2.

**Comment by PeterMeisrimelModelon** [comment]
This should now be fixed in the 3.0a2 release
